pop-api/src/v0/assets/fungibles.rs

```diff
@@ -323,9 +324,46 @@ pub fn burn(id: AssetId, account: AccountId, value: Balance) -> Result<()> {
 		.call(&(id, account, value))
 }
 
+/// The interface for managing metadata of fungible assets. It includes the PSP-22 Metadata interface
+/// for querying metadata.
+// TODO: if reviewers agree with replacing metadata related apis here, the dispatchable index has to
```

Comment:
Seems to sit better in relation to the PSP22 standard. Not sure about the `dispatchable index` changing, figure it may be something to do with the ordering of dispatchables to align with the standard? As they are not part of the standard, does the index matter?

Reply:
Changed back. 

As for the index, it is just based on the order of the functions in the file.